SECURITY_CONTACTS

```diff
@@ -0,0 +1,15 @@
+# Reporting a security vulnerability
```

Comment:
We should exclude this file to avoid confusion.

Reply:
Addressed in #253